src/modules/documentExecution/dashboard/DashboardHelpers.ts

```diff
@@ -19,7 +19,7 @@ const store = mainStore()
 const SIZES = ['xxs', 'xs', 'sm', 'md', 'lg'] as string[]
 
 export const createNewDashboardModel = () => {
-    const dashboardModel = descriptor.newDashboardModel as IDashboard
+    const dashboardModel = deepcopy(descriptor.newDashboardModel) as IDashboard
```

Comment:
deepcopy added to be sure object are different and not to use the same dashboardModel on different new dashboards